src/instructlab/config/init.py

```diff
@@ -72,61 +74,80 @@ def init(
 ):
     """Initializes environment for InstructLab"""
     ensure_storage_directories_exist()
-    clone_taxonomy_repo = True
-    if interactive:
-        if exists(DEFAULTS.CONFIG_FILE):
-            overwrite = click.confirm(
-                f"Found {DEFAULTS.CONFIG_FILE} in the current directory, do you still want to continue?"
-            )
-            if not overwrite:
-                return
-        click.echo(
-            "Welcome to InstructLab CLI. This guide will help you to setup your environment."
-        )
-        click.echo(
-            "Please provide the following values to initiate the "
-            "environment [press Enter for defaults]:"
-        )
-        taxonomy_path = utils.expand_path(
-            click.prompt("Path to taxonomy repo", default=taxonomy_path)
-        )
+    cfg = None
+    # If the caller used `ilab init` then the parent command is `ilab`, so we need to get the
+    # parameter source from the parent command. If the caller used `ilab config init` then the
+    if ctx.parent.command.name == "ilab":
```

Comment:
correct!

Reply:
also I think after speaking w/ @mairin we want running something like `ilab model chat` to just _work_ if you have the `ILAB_GLOBAL_CONFIG` env var set for those types of users. So technically we will tell them to run `ilab config init` but if they skip it, it will pull from the golden cfg? 

This may be confusing though